features/groups.feature

```diff
@@ -152,8 +152,8 @@ Feature:  groups
       end
 
       case Shutters
-      when UP then logger.info('All shutters are UP')
-      when DOWN then logger.info('At least one shutter is not UP')
+      when 0 then logger.info('All shutters are UP')
+      when 100 then logger.info('At least one shutter is not UP')
```

Comment:
This part here is a bit counter-intuitive. The comparison in a case statement for item `state` (as opposed to `command`) also requires an exact type match, in this case, because the state's type is PercentType, it must be compared against PercentType.

Reply:
Does that need to be in the docs so we and others don't get confused?

Reply:
> Does that need to be in the docs so we and others don't get confused?

Ideally, we'll have a page that comprehensively explains about how state comparisons work in various different ways. For example, how and why PercentType.new(1).on? returns true vs PercentType.new(1) === ON returns false. 

I'll create a separate PR for that because I may not think of all possible scenarios.